content/hi/docs/concepts/architecture/cloud-controller.md

```diff
@@ -0,0 +1,215 @@
+---
+title: क्लाउड नियंत्रक प्रबंधक
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+{{< feature-state state="beta" for_k8s_version="v1.11" >}}
+
+क्लाउड इंफ्रास्ट्रक्चर टेक्नोलॉजी आपको सार्वजनिक, निजी और हाइब्रिड क्लाउड पर कुबेरनेट चलाने देती है।
+कुबेरनेट्स स्वचालित, एपीआई-संचालित बुनियादी ढांचे के बीच तंग युग्मन के बिना विश्वास करता है
+अवयव।
+
+{{< glossary_definition term_id="cloud-controller-manager" length="all" prepend="The cloud-controller-manager is">}}
+
+क्लाउड-नियंत्रक-प्रबंधक को एक प्लगइन का उपयोग करके संरचित किया जाता है
+तंत्र जो विभिन्न क्लाउड प्रदाताओं को कुबेरनेट्स के साथ अपने प्लेटफॉर्म को एकीकृत करने की अनुमति देता है।
+
+<!-- body -->
+
+## डिज़ाइन
+
+![कुबेरनेट्स घटक](/images/docs/components-of-kubernetes.svg)
+
+क्लाउड कंट्रोलर मैनेजर कंट्रोल प्लेन में प्रक्रियाओं के एक प्रतिकृति सेट के रूप में चलता है
+(आमतौर पर, ये पॉड्स में कंटेनर होते हैं)। प्रत्येक क्लाउड-नियंत्रक-प्रबंधक लागू करता है
+एकाधिक {{< glosary_tooltip text="controllers" term_id="controller" >}} एक में
+प्रक्रिया।
+
+
+{{< नोट >}}
+आप क्लाउड कंट्रोलर मैनेजर को कुबेरनेट्स के रूप में भी चला सकते हैं
+{{< glosary_tooltip text="addon" term_id="addons" >}} भाग के बजाय
+नियंत्रण विमान का।
+{{< /नोट >}}
+
+## क्लाउड नियंत्रक प्रबंधक कार्य करता है {#फ़ंक्शंस-ऑफ़-द-सीसीएम}
+
+क्लाउड नियंत्रक प्रबंधक के अंदर नियंत्रकों में शामिल हैं:
+
+### नोड नियंत्रक
+
+नोड नियंत्रक {{< glosary_tooltip text="Node" term_id="node" >}} वस्तुओं को अद्यतन करने के लिए जिम्मेदार है
+जब आपके क्लाउड इन्फ्रास्ट्रक्चर में नए सर्वर बनाए जाते हैं। नोड नियंत्रक के बारे में जानकारी प्राप्त करता है
+क्लाउड प्रदाता के साथ आपकी किरायेदारी के भीतर चल रहे होस्ट। नोड नियंत्रक निम्नलिखित कार्य करता है:
+
+1. क्लाउड प्रदाता एपीआई से प्राप्त संबंधित सर्वर के अद्वितीय पहचानकर्ता के साथ नोड ऑब्जेक्ट को अपडेट करें।
+2. नोड ऑब्जेक्ट को क्लाउड-विशिष्ट जानकारी के साथ एनोटेट करना और लेबल करना, जैसे कि क्षेत्र नोड
+   उपलब्ध संसाधनों (सीपीयू, मेमोरी, आदि) में तैनात है।
+3. नोड का होस्टनाम और नेटवर्क पता प्राप्त करें।
+4. नोड के स्वास्थ्य का सत्यापन करना। यदि कोई नोड अनुत्तरदायी हो जाता है, तो यह नियंत्रक जाँच करता है
+   आपके क्लाउड प्रदाता की एपीआई यह देखने के लिए कि क्या सर्वर को निष्क्रिय/हटाया/समाप्त कर दिया गया है।
+   यदि नोड को क्लाउड से हटा दिया गया है, तो नियंत्रक आपके कुबेरनेट्स से नोड ऑब्जेक्ट को हटा देता है
+   झुंड।
+
+कुछ क्लाउड प्रदाता कार्यान्वयन इसे एक नोड नियंत्रक और एक अलग नोड में विभाजित करते हैं
+जीवनचक्र नियंत्रक।
+
+### मार्ग नियंत्रक
+
+रूट कंट्रोलर क्लाउड में रूट्स को कॉन्फ़िगर करने के लिए जिम्मेदार है
+उचित रूप से ताकि आपके कुबेरनेट्स में विभिन्न नोड्स पर कंटेनर हों
+क्लस्टर एक दूसरे के साथ संवाद कर सकते हैं।
+
+क्लाउड प्रदाता के आधार पर, रूट कंट्रोलर ब्लॉक भी आवंटित कर सकता है
+पॉड नेटवर्क के लिए IP पतों की संख्या।
+
+### सेवा नियंत्रक
+
+{{< glosary_tooltip text="Services" term_id="service" >}} क्लाउड के साथ एकीकृत
```

Comment:
Typo in the `glossary_tooltip` shortcode name. Also, localize the "text" parameter passed to the shortcode to Hindi.